vendor/github.com/smartystreets/assertions/.travis.yml

```diff
@@ -2,11 +2,10 @@ language: go
 
 go:
   - 1.x
-  - master
```

Comment:
Why is this stuff committed in the vendor directory?

Reply:
this part is weird. i had no say in this. i simply ran `dep ensure`